rl_games/common/a2c_common.py

```diff
@@ -966,7 +973,7 @@ def train(self):
                 update_time = 0
             if self.multi_gpu:
                     should_exit_t = torch.tensor(should_exit).float()
-                    self.hvd.broadcast_value(should_exit_t, 'should_exit')
+                    # self.hvd.broadcast_value(should_exit_t, 'should_exit') # what is the purpose of this?
```

Comment:
here is a chance that one job will exit  a little bit earlier, as result other jobs will crash.

Reply:
This method seems to be deprecated by horovod: the closest method I can find is https://horovod.readthedocs.io/en/stable/api.html#horovod.torch.broadcast_. 

What exactly is this `broadcast_value` doing? If rank 0's `should_exit_t=False` and rank 1's `should_exit_t=True`, would `broadcast_value` overwrite rank 1's `should_exit_t`?